gm4_animi_shamir/beet.yaml

```diff
@@ -20,6 +20,7 @@ meta:
       required:
         gm4_metallurgy: 1.4.0
         lib_player_death: 1.2.0
+      environment_checks: [gm4:score_on_non_player_entity]
```

Comment:
What was your reasoning for making this configurable? Is there any reason we wouldn't want to just check the environment in every module, especially since they all use scoreboards to initialize through lantern load.

Reply:
That's mostly true for this check, but not for all. E.g. `lib_machines` need command blocks to be enabled, but most modules don't need that.